src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurly.java

```diff
@@ -4,7 +4,8 @@ public class InputRightCurly {
     public static void main(String[] args) {
         boolean after = false;
         try {
-        } finally {
+        } // violation ''}' at column 9 should be on the same line as the next part'
+        finally {
```

Comment:
Restored the old curly location and added the violation message

Reply:
@Zopsss , please restore as it was original and just place a comment if there is violation.

Reply:
Original had no violation, so if there is one now that is a difference and the line isn't restored.

Reply:
corrected the restored code. The violation is occurring because of `{`

https://github.com/checkstyle/checkstyle/blob/148e78d25899444ba5531ff03f1db5078af7a899/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurly.java#L6-L8

It was not occurring before because we were using the `RightCurly` module only. In the new test, we're using both module: `LeftCurly` & `RightCurly` so it became violation for LeftCurly

Reply:
> LeftCurly & RightCurly so it became violation for LeftCurly

This makes sense, but this still seems a bit odd. Why is there a violation for left curly but right curly is saying everything is ok?

I think we need to review this and the style guide. This looks like it might be a bug of some kind.

Reply:
I am going to approve this. I still think it is odd Right is fine.